### ISSUE TITLE
Cannot change the MEMORY_MAPPING_ENGINE

### ISSUE THREAD

**Issue by minouHub** [radis/radis#401]
### 🐛 Describe the bug
Hello,

I tried running RADIS 0.11, changing the `radis.json` file with

- `"MEMORY_MAPPING_ENGINE": "pytables"`

- and `"MEMORY_MAPPING_ENGINE": "feathers"`

### 🖥 Steps To Reproduce

```python
from radis import config
config["MEMORY_MAPPING_ENGINE"] = "pytables" #EDIT that potentially fix the error


from radis import calc_spectrum
s = calc_spectrum(
            1900,
            2300,  # cm-1
            molecule="CO",
            isotope="1,2,3",
            pressure=1.01325,  # bar
            Tgas=700,  # K
            mole_fraction=0.1,
            path_length=1,  # cm
            databank="hitran",  # or use 'hitemp'
    
)
```
The test didn't run because these formats are not implemented, see this portion of calc.py

```python
def get_filenames(self):
        if self.engine == "vaex":
            return [join(self.local_databases, f"{self.molecule}.hdf5")]
        else:
            raise NotImplementedError()
```

-------------------------
Another test failed with the same config file, HITEMP this time, and crashed in the conversion:

```python
s = calc_spectrum(1900, 2300,         # cm-1
                  molecule='CO',
                  isotope='1,2,3',
                  pressure=1.01325,   # bar
                  Tgas=700,           # K
                  mole_fraction=0.1,
                  path_length=1,      # cm
                  databank='hitemp',  # or use 'hitemp'
                  )
```
```python
Downloading 05_HITEMP2019.par.bz2 for CO (1/1).
Download complete. Parsing CO database to C:\Users\mines\.radisdb\hitemp\CO-05_HITEMP2019.h5
Traceback (most recent call last):

  File "<ipython-input-5-f94317cc3f69>", line 2, in <module>
    s = calc_spectrum(1900, 2300,         # cm-1

  File "e:\python\radis\radis\lbl\calc.py", line 508, in calc_spectrum
    generated_spectrum = _calc_spectrum_one_molecule(

  File "e:\python\radis\radis\lbl\calc.py", line 721, in _calc_spectrum_one_molecule
    sf.fetch_databank(**conditions)

  File "e:\python\radis\radis\lbl\loader.py", line 1194, in fetch_databank
    df, local_paths = fetch_hitemp(

  File "e:\python\radis\radis\io\hitemp.py", line 577, in fetch_hitemp
    ldb.download_and_parse(download_urls, download_files)

  File "e:\python\radis\radis\io\dbmanager.py", line 370, in download_and_parse
    download_and_parse_one_file(urlname, local_file, Ndownload)

  File "e:\python\radis\radis\io\dbmanager.py", line 335, in download_and_parse_one_file
    Nlines = self.parse_to_local_file(

  File "e:\python\radis\radis\io\hitemp.py", line 345, in parse_to_local_file
    writer.write(local_file, df, append=True)

  File "e:\python\radis\radis\io\hdf5.py", line 170, in write
    raise NotImplementedError(self.engine)

NotImplementedError: feathers
```

### 💡 Possible solutions
I guess @erwanp you removed some portions of codes from v0.10 that were not meant to be?

**Comment by minouHub** [comment]
Hello Erwan,

The fix sorted HITRAN and HITEMP calcs, but not ExoMol. The process was stuck again, and I could unstuck it by running a new console. Then I got this error. After checking my radisdb/exomol, it appeared the code was still converting exomol to `.h5` and not `.feather`.

```python
radis.config['MEMORY_MAPPING_ENGINE'] = 'feather'
s = calc_spectrum(1900, 2300,         # cm-1
                  molecule='CO',
                  isotope='1,2,3',
                  pressure=1.01325,   # bar
                  Tgas=700,           # K
                  mole_fraction=0.1,
                  path_length=1,      # cm
                  databank='exomol',  # or use 'hitemp'
                  )
```

```python

  File "E:\Python\bug_radis\version_0.11.py", line 45, in <module>
    s = calc_spectrum(1900, 2300,         # cm-1

  File "e:\python\radis\radis\lbl\calc.py", line 507, in calc_spectrum
    generated_spectrum = _calc_spectrum_one_molecule(

  File "e:\python\radis\radis\lbl\calc.py", line 720, in _calc_spectrum_one_molecule
    sf.fetch_databank(**conditions)

  File "e:\python\radis\radis\lbl\loader.py", line 1252, in fetch_databank
    df, local_path, Z_exomol = fetch_exomol(

  File "e:\python\radis\radis\io\exomol.py", line 362, in fetch_exomol
    mdb = MdbExomol(local_path, [load_wavenum_min, load_wavenum_max], engine=engine)

  File "e:\python\radis\radis\io\exomol.py", line 539, in __init__
    ndstates.to_feather(self.states_file.with_suffix(".feather"))

  File "C:\Users\mines\anaconda3\lib\site-packages\vaex\dataframe.py", line 260, in __getattr__
    return object.__getattribute__(self, name)

AttributeError: 'DataFrameLocal' object has no attribute 'to_feather'
```

**Comment by erwanp** [comment]
Starting from Vaex 4.8 there are no more compatibility problem with Spyder. 

Fix of #402 is removed in Radis 0.13
